Azure/sonic-frr
# Support SONiC debian8 on FRR 5.0.x

**- What I did**
Make FRR 5.0 buildable with SONiC debian8 dev environment. 

**- How I did it**
Add sonic.debian8 target with proper settings for SONiC

**- How to verify it**
Build with sonic-buildimage (PR TBD), and load the image.  FRR starts fine, and also FPM worked as expected towards SONiC :

$ vtysh

Hello, this is FRRouting (version 5.0.1).
Copyright 1996-2005 Kunihiro Ishiguro, et al.

sonic# show running-config 
Building configuration...

Current configuration:
!
frr version 5.0.1
frr defaults traditional
hostname sonic
log syslog informational
log facility local4
service integrated-vtysh-config
username cumulus nopassword
!
password zebra
enable password zebra
!
router bgp 65100
 bgp router-id 10.1.0.1
 bgp log-neighbor-changes
 no bgp default ipv4-unicast
 bgp bestpath as-path multipath-relax
 neighbor 10.0.0.1 remote-as 65200
 neighbor 10.0.0.1 description ARISTA01T2
 neighbor 10.0.0.3 remote-as 65200
 neighbor 10.0.0.3 description ARISTA02T2
 neighbor 10.0.0.5 remote-as 65200
 neighbor 10.0.0.5 description ARISTA03T2
 neighbor 10.0.0.7 remote-as 65200
 
 ....
 
sonic# configure terminal 
sonic(config)# ip route 10.10.10.0/24 10.0.0.17

redis-cli:
 
 127.0.0.1:6379> hgetall ROUTE_TABLE:10.10.10.0/24
1) "nexthop"
2) "10.0.0.17"
3) "ifname"
4) "Ethernet8"


**- Description for the changelog**
